docs-src/cloud/namespaces-create.md

```diff
@@ -18,7 +20,7 @@ To create a Namespace, a user must have the Developer or Global Admin account-le
 :::tip
 
 By default, each account has a quota of 10 Namespaces.
-If you want to increase this limit, open a [support ticket](https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support).
+If you want to increase this limit, open a [support ticket](https://docs.temporal.io/cloud/support-create-ticket).
```

Comment:
```suggestion
If you want to increase this limit, open a [support ticket](https://docs.temporal.io/cloud/introduction/support#support-ticket).
```

Reply:
this should actually be a id and then link magic will catch it.